### ISSUE TITLE
'High Contrast Black' theme Tabs selected text color contrast

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1018]
**Describe the bug**
The Tabs selected text color contrast ratio is less than 4.5:1
Also for selected/active tabs, fd-tabs__icon text color should be in line with the theme i.e black.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/tabs.html
2. Check the Tabs selected text color contrast ratio

**Expected behavior**
The Tabs selected text color contrast ratio should be 4.5:1 or above. Ideally 7:1 or above for WCAG AAA compliance. #3599FD would be the best choice.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81759590-f7594d80-9479-11ea-9c88-eb81a64c44ad.png)
![image](https://user-images.githubusercontent.com/43764832/81759613-06400000-947a-11ea-8da3-d40a5e52cc57.png)
![image](https://user-images.githubusercontent.com/43764832/81865105-5b375100-9522-11ea-8854-cdcb027779e4.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 19.6:1
<img width="227" alt="Screen Shot 2020-10-07 at 5 55 32 PM" src="https://user-images.githubusercontent.com/21978807/95530285-59b76b00-0992-11eb-8a80-efd32e604942.png">
<img width="301" alt="Screen Shot 2020-10-08 at 6 17 04 PM" src="https://user-images.githubusercontent.com/21978807/95530351-7e134780-0992-11eb-917f-5397d760726c.png">

**Comment by prsdthkr** [comment]
Check with HCD Theme selected:
https://deploy-preview-1752--fundamental-styles.netlify.app/?path=/docs/components-tabs--primary
